holoviews/plotting/mpl/raster.py

```diff
@@ -150,6 +150,9 @@ class RasterGridPlot(GridPlot, OverlayPlot):
     equivalent using subplots.
     """
 
+    padding = param.Number(default=0.1, doc="""
+        The amount of padding as a fraction of the total Grid size""")
```

Comment:
This `padding` parameter is a bit different from the common one...

Reply:
Yes, thanks for catching that. It might cause confusion but I don't think it's a huge issue, it's on ``GridPlot`` and therefore won't actually clash, and also only exists in the matplotlib backend.